frontend/src/scenes/billing/BillingProductAddon.tsx

```diff
@@ -237,6 +303,57 @@ export const BillingProductAddon = ({ addon }: { addon: BillingProductV2AddonTyp
                 }
             />
             {surveyID && <UnsubscribeSurveyModal product={addon} />}
+            <LemonModal
```

Comment:
I don't think we should do this in a modal, it's an unnecessary extra step. Can we put the required info in the main pane?

Reply:
I disagree. We've seen so many people expect something to happen when they click the button on the add-on, and there's really too much information to show on the add-on card. I think the move to a trial warrants the extra step to confirm.

Reply:
I thought we were doing the trial as a way to avoid the extra modal though. Because the modal prevented conversion in the past. Since it's a no-risk click then there's no downside in it happening automatically (unless they get grumpy because they want to start the trial later - but we can always handle that..)

Reply:
I don't think we can clearly show enough information on the card without making it blotted. I tried a few things but couldn't make it work. I also still think the double click makes sense, even with the trial. It feels weird in the UX to click a button, and it just happens without confirming when it's billing-related, something I realized while building this. I could A/B this if you're worried about the conversion.

Reply:
I'll A/B test it and see how conversion differs.